### ISSUE TITLE
Broken Pipe Error after installing

### ISSUE THREAD

**Issue by ezekini-uala** [IlanCosman/tide#143]
#### Describe the bug

After installing the package I started getting this message when opening the terminal (terminal or iterm):
```
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
```
As far as I can see, everything works fine, but couldn't finde the solution on my own.

#### Steps to reproduce

1. Install fish
2. Install fisher
3. Install tide

#### Environment

- Operating system: Mac Os 10.15.5
- Terminal emulator: Iterm2

**Comment by IlanCosman** [comment]
Sorry for the late reply. I think I've seen this once before, it has to do with writing to the universal variable file, but I don't know much more 😞

**Comment by zyddnys** [comment]
Same here on Ubuntu 20.04

**Comment by IlanCosman** [comment]
Closing as stale; Fish 3.3 released and almost no one has reported this issue.

**Comment by pwwang** [comment]
Same here on Ubuntu 20.04 WSL2, fish 3.3.1

**Comment by chstan** [comment]
Can also confirm on Ubuntu 20.04, fish 3.3.1.

**Comment by pwwang** [comment]
This can be reproduced by fast tapping some ENTERs.

**Comment by smthpickboy** [comment]
> @smthpickboy What's the difference by putting `fisher update ilanCosman/tide` instead of doing it manually?
> 
> I updated to 5.0.1 manually, the issue still persists.

No difference. But when the problem happened, my shell simply couldn't start. So I have to put it in config.fish.

**Comment by IlanCosman** [comment]
Are any of you using Conda?

**Comment by ezekini-uala** [comment]
> Are any of you using Conda?

I have miniconda installed

**Comment by pwwang** [comment]
I am using conda, too.

**Comment by IlanCosman** [comment]
Alright yeah, this bug is coming from my ancient nemesis Conda 😂  I'll see if I can do anything about it.

**Comment by abhishek-1131** [comment]
> This can be reproduced by fast tapping some ENTERs.

Facing the same issue, any solutions yet? Thanks!

**Comment by IlanCosman** [comment]
Yes, you can solve this problem by adding `status is-interactive &&` before the conda init:

```fish
# >>> conda initialize >>>
# !! Contents within this block are managed by 'conda init' !!
status is-interactive && eval /Users/ilan/opt/anaconda3/bin/conda "shell.fish" "hook" $argv | source
# <<< conda initialize <<<
```
